panel/src/libraries/dayjs.js

```diff
@@ -1,5 +1,6 @@
 import dayjs from "dayjs";
 import customParseFormat from "dayjs/plugin/customParseFormat";
+import relativeTime from "dayjs/plugin/relativeTime";
```

Comment:
`Concern:` We won't have any language support for this, will we? Cause in that case we will get reports from users right away. Not sure it's worth it then.

Reply:
You are right! Maybe let's go with a simple date and time solution first.